### PR TITLE
Plugins

### PR DESCRIPTION
**Pull request by Rider21**
rebuilt foxaholic, LightNovelUpdates plugins in madara
fixed the readwn search

### CODE REVIEW

**Comment by nyagami** [comment]
If you move some plugins into multisrc, pls delete their old .js files in .js dir
quick way: `rm -r .js && tsc`

File: icons/multisrc/madara/icons/lightnovelupdates.png
Comment (K1ngfish3r) [inline icons/multisrc/madara/icons/lightnovelupdates.png:1]: lightnovelupdates
i make smaller for you

**Comment by K1ngfish3r** [comment]
> I think the TuNovelaLigera plugin is also related to madara?

The madara site design is just a cover up at this point (empty chapters if you click on them), actual chapters are elsewhere

File: icons/multisrc/madara/icons/lightnovelupdates.png
Comment (Rider21) [reply icons/multisrc/madara/icons/lightnovelupdates.png:1]: thanks

**Comment by nyagami** [comment]
Sorry, I've just updated repo, and now plugins are hosted in `dist` branch. Please resolve the conflict

**Comment by Rider21** [comment]
added sources WuxiaMTL, FanNovel (closes #688)

File: plugins/russian/LitSpace.ts [hunk 1]
```diff
@@ -1,241 +1,242 @@
...
+    };
+
+    const chapters: Plugin.ChapterItem[] = [];
+
+    loadedCheerio("a.chapter-line").each((chapterIndex, element) => {
+      const name = loadedCheerio(element).find("h6").text();
+      const url = loadedCheerio(element).attr("href");
+      if (!name || !url) return;
+
+      const releaseTime = loadedCheerio(element).find('span[class="date"]').text();
```
Comment (nyagami) [inline plugins/russian/LitSpace.ts:78]: could you parse this into ISO string format.

File: plugins/russian/authortoday.ts [hunk 2]
```diff
@@ -1,538 +1,529 @@
...
+    });
+
+    const chaptersJSON = (await chaptersRaw.json()) as ChaptersEntity[];
+    const chapters: Plugin.ChapterItem[] = [];
+    chaptersJSON.forEach((chapter, chapterIndex) => {
+      if (chapter.isAvailable && !chapter.isDraft) {
+        chapters.push({
+          name: chapter.title || 'Глава ' + (chapterIndex + 1),
+          url: `${apiUrl}v1/work/${workID}/chapter/${chapter.id}/text`,
+          releaseTime: dayjs(chapter.publishTime || chapter.lastModificationTime).format("LLL"),
```
Comment (nyagami) [inline plugins/russian/authortoday.ts:104]: could you check if this work in app? hermes engine is stupid with some date format.

Comment (Rider21) [reply plugins/russian/authortoday.ts:104]: It's working. 
Screenshot_20240204-075331
Screenshot_20240204-075350_1

File: plugins/russian/LitSpace.ts (hunk 1 above)
Comment (Rider21) [reply plugins/russian/LitSpace.ts:78]: Perhaps it would be better to pass an instance of Date?

**Review by nyagami — APPROVED** [review]

File: plugins/russian/LitSpace.ts (hunk 1 above)
Comment (nyagami) [reply plugins/russian/LitSpace.ts:78]: it's not mandatory, we just want a consistent Date system xD

**Pull request merged**